Adjusting the documentation - 1.5 installing git - `install-info` package

Small suggestion about compiling git documentation.
URL: https://git-scm.com/book/en/v2/Getting-Started-Installing-Git

**"Installing from Source" part:**

> $ make all doc info
> $ sudo make install install-doc install-html install-info

----

On a Debian Stretch 9.4 fresh install:
```
# make install install-doc install-html install-info
[...]
install -d -m 755 /usr/share/info
install -m 644 git.info gitman.info /usr/share/info
if test -r /usr/share/info/dir; then \
  install-info --info-dir=/usr/share/info git.info ;\
  install-info --info-dir=/usr/share/info gitman.info ;\
else \
  echo "No directory found in /usr/share/info" >&2 ; \
fi
/bin/sh: 2: install-info: not found
/bin/sh: 3: install-info: not found
Makefile:259: recipe for target 'install-info' failed
make[1]: *** [install-info] Error 127
make[1]: Leaving directory '~/git.git/Documentation'
Makefile:2809: recipe for target 'install-info' failed
make: *** [install-info] Error 2
```

After running `apt install install-info` and re-run the make install command, everything works fine:
```
# git --version
git version 2.17.0.582.gccdcbd54c
```


- It might be usefull to adjust the documentation to add the installation of `install-info` package?

Thanks for your awesome work anyway, great doc.

--- a/book/01-introduction/sections/installing.asc
+++ b/book/01-introduction/sections/installing.asc
@@ -91,8 +91,8 @@ In order to be able to add the documentation in various formats (doc, html, info
 
 [source,console]
 ----
-$ sudo dnf install asciidoc xmlto docbook2X getopt
-$ sudo apt-get install asciidoc xmlto docbook2x getopt
+$ sudo dnf install asciidoc xmlto docbook2X
+$ sudo apt-get install asciidoc xmlto docbook2x
 ----
 
 If you're using a Debian-based distribution (Debian/Ubuntu/Ubuntu-derivatives), you also need the `install-info` package:

--- a/book/01-introduction/sections/installing.asc
+++ b/book/01-introduction/sections/installing.asc
@@ -95,6 +95,21 @@ $ sudo dnf install asciidoc xmlto docbook2X getopt
 $ sudo apt-get install asciidoc xmlto docbook2x getopt
 ----
 
+If you're using a Debian-based distribution (Debian/Ubuntu/Ubuntu-derivatives), you also need the `install-info` package:
+
+[source,console]
+----
+$ sudo apt-get install install-info
+----
+
+If you're using a RPM-based distribution (Fedora/RHEL/RHEL-derivatives), you also need the `getopt` package (which is already installed on a Debian-based distro):
+
+[source,console]
+----
+$ sudo dnf install getopt
+$ sudo apt-get install getopt
+----
+
 Additionally, if you're using Fedora/RHEL/RHEL-derivatives, you need to do this
 
 [source,console]

